backend/api/Areas/Contacts/Models/Contact/OrganizationModel.cs

```diff
@@ -7,39 +9,49 @@ public class OrganizationModel
     {
         #region Properties
         /// <summary>
-        /// get/set - The primary key to identify the organization.
+        /// get/set - The organization's id.
         /// </summary>
         public long Id { get; set; }
 
         /// <summary>
-        /// get/set - The organization's company name.
+        /// get/set - The organization's disabled status flag.
+        /// </summary>
+        public bool IsDisabled { get; set; }
+
+        /// <summary>
+        /// get/set - The organization's concatenated name.
         /// </summary>
         public string Name { get; set; }
 
         /// <summary>
-        /// get/set - The organization's primary contact name.
+        /// get/set - The organization's alias name.
+        /// </summary>
+        public string Alias { get; set; }
+
+        /// <summary>
+        /// get/set - The organization's incorporation number.
         /// </summary>
-        public string ContactName { get; set; }
+        public string IncorporationNumber { get; set; }
 
         /// <summary>
-        /// get/set - The organizations's email.
+        /// get/set - The organization's organizations.
```

Comment:
Probably meant the organization's persons.

Reply:
yep